test/integration/test_frame.py

```diff
@@ -0,0 +1,55 @@
+import os
+
+import pytest as pytest
+
+from FLIR.conservator.generated.schema import AnnotationCreate
+from FLIR.conservator.util import md5sum_file
+
+
+def test_add_annotation(conservator, test_data):
+    path = test_data / "jpg" / "bottle_0.jpg"
+    media_id = conservator.media.upload(path)
+    conservator.media.wait_for_processing(media_id, check_frequency_seconds=1)
+    image = conservator.images.all().first()
+    frame = image.get_frame()
+
+    frame.populate(["annotations", "annotations_count"])
+    assert len(frame.annotations) == 0
+    assert frame.annotations_count == 0
+
+    annotation_create = AnnotationCreate(
+        labels=["bottle", "cat"], bounding_box={"x": 1, "y": 2, "w": 3, "h": 4}
+    )
+    frame.add_annotation(annotation_create)
+
+    frame.populate(["annotations", "annotations_count"])
+    assert len(frame.annotations) == 1
+    assert frame.annotations_count == 1
+    annotation = frame.annotations[0]
+    assert annotation.bounding_box.x == 1
+    assert annotation.bounding_box.y == 2
+    assert annotation.bounding_box.w == 3
+    assert annotation.bounding_box.h == 4
+    assert annotation.labels == ["bottle", "cat"]
+    assert annotation.source.type == "human"
+
+
+@pytest.mark.skip()
+def test_add_prediction(conservator, test_data):
+    # There are no classifiers, so this isn't trivial to test.
+    # Skipping until someone actually uses it and/or finds a bug.
+    pass
```

Comment:
I believe there is a fixed set of prediction IDs hardcoded into flirconservator.

Reply:
Does the query in the method here work in a new local flirconservator instance: https://github.com/FLIR/ConservatorTools/blob/3794551fdcdab438184c412f933ef0b7462b0c22/FLIR/ConservatorTools/common/conservator_operations.py#L33

Reply:
Yup, that works. Added!